includes/definitions/ecreso-fm-tx.yaml

```diff
@@ -0,0 +1,28 @@
+os: worldcastsystems
+text: "Ecreso FM Transmitter"
+type: transmitter
+group: ecreso
+icon: worldcastsystems_square
+over:
+  - { graph: device_power, text: Power }
+  - { graph: device_voltage, text: Voltage }
+  - { graph: device_count, text: Count }
+discovery:
+  - sysObjectID:
+      - .1.3.6.1.4.1.8072.3.2.10
```

Comment:
This is a linux sysObjectID. You will need to use sysDescr (which doesn't look too helpful here) or snmpget an old only known to this vendor, like: `includes/definitions/quantastor.yaml`